extensions/wikia/CreateNewWiki/css/_variables.scss

```diff
@@ -2,6 +2,7 @@ $content-spacing-xxsmall: 6px;
 $content-spacing-xsmall: 10px;
 $content-spacing-small: 12px;
 $content-spacing-small-plus: 14px;
+$content-spacing-regular-minus: 16px;
```

Comment:
Those names are used the same way in CommunityPage. Please add a new variable instead of changing their sizes to avoid confusion.